argoverse/data_loading/frame_label_accumulator.py

```diff
@@ -65,7 +62,6 @@ def __init__(
         log_egopose_dict_fpath = f"{tmp_dir}/log_egopose_dict_{experiment_prefix}.pkl"
         log_timestamp_dict_fpath = f"{tmp_dir}/log_timestamp_dict_{experiment_prefix}.pkl"
 
-        coordinate_system = "map_world_fr"
```

Comment:
changes look good. can we make this a comment though? that `# coordinate system is the map world frame`

Reply:
Done 👍